frontend/src/pages/add-blog.tsx

```diff
@@ -129,17 +129,8 @@ function AddBlog() {
           />
         </div>
 
-        <div className="mb-4">
```

Comment:
Why have you removed description from here ?

Reply:
sir I removed description from here because it is givn in your figma design

![figma](https://github.com/krishnaacharyaa/wanderlust/assets/110713125/2da03f9c-cbcd-4bda-9c39-0d2d9ccbdff2)

Reply:
Ooh my bad, I'll change that in the figma file, lets keep it original to the code , description and then categories.

Reply:
sir it look great to first keep categories first User can write description by keeping category in mind